Week2/homework/js-exercises/recipe-ex3.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+let recipe = {
+    mealName : "Mirzaghasemi",
+    serves: 2,
+    ingredients : ["2 eggplants", "2 eggs", "1 garlic"]
+};
+//Method1:
+// for (let property in recipe) {
+//     console.log(property + ": " + recipe[property]);
```

Comment:
the ordering of lines in the output might be different in other browsers. You should be careful if ordering matters. or you need a different solution.

By the way you solutions are hacky. your output should be:

Meal name: ...
Serves: ..
Ingredients: ...

yours are just keys of the dictionary. please adjust it